src/ATen/DLConvertor.h

```diff
@@ -10,8 +10,8 @@
 
 namespace at {
 
-ScalarType toScalarType(const DLDataType& dtype);
-DLManagedTensor * toDLPack(const Tensor& src);
-Tensor fromDLPack(const DLManagedTensor* src);
+ATen_CLASS ScalarType toScalarType(const DLDataType& dtype);
```

Comment:
why are these decorators needed? I don't have much context on this.

Reply:
To export these functions, i think. If not added, the compiling using MSVC will fail.

Reply:
Additional information here:
> If you use other compilers like gcc on Linux or MinGW gcc compiler on Windows you don't have linking errors due to not exported symbols, because gcc compiler export all symbols in a dynamic library (dll) by default instead of MSVC or Intel windows compilers.
>
> In windows you have to explicitly export symbol from a dll.

Reply:
thanks for the context. Is there a decorator for functions something like `ATen_FUNC` rather than `ATen_CLASS`?  but otherwise sounds good to me.

Reply:
I think ATen_CLASS is confusing when it is being used on functions. Can macro be renamed `AT_API` (`AT` to follow the other macro conventions)? The current `ATen_API` macros are unused and can be removed -- we don't export any C functions.